packages/material-ui/src/TextField/TextField.d.ts

```diff
@@ -93,7 +93,7 @@ export interface BaseTextFieldProps
   /**
    * If `dense` or `normal`, will adjust vertical spacing of this and contained components.
    */
-  margin?: PropTypes.Margin;
+  margin?: 'none' | 'dense' | 'normal';
```

Comment:
Why we did these changes?

Reply:
Hi @mnajdova I did not did this particular change, I consider   
```
margin?: PropTypes.Margin;	
```
complies but @oliviertassinari may clear this up

Reply:
I have removed the `PropTypes` types as it was introduced to enforce more consistency between different components. However, it appears that it's barely used anywhere else, so for simplicity, I went with duplication.

Reply:
Makes sense, thanks for the clarification